FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
But if people have VS2015, then open the solution, edit a file, and start using `nameof`, they'll be all excited and happy and then later on `rake` will fail (or worse the CI build for the PR), and then frustration…